packages/@ember/deprecated-features/index.ts

```diff
@@ -0,0 +1 @@
+export const DESCRIPTOR_VALUES = '3.2.0';
```

Comment:
Perhaps we could go with `DESCRIPTOR_FALLBACK_PROPERTY_LOOKUP` or `PROPERTY_BASED_DESCRIPTORS`?